.github/workflows/ci.yml

```
@@ -13,7 +13,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: [3.7, 3.8]
+        python-version: [3.7, 3.8, 3.9, 3.10]
```

```suggestion
        python-version: [3.7, 3.8, 3.9, '3.10']
```
See: https://github.com/actions/setup-python/issues/160#issuecomment-724485470